war/src/main/less/abstracts/theme.less

```
@@ -280,7 +280,7 @@
   --menu-text-color: black;
   --menu-bg-color: var(--white);
   --menu-selected-color: #b3d4ff;
-  --menu-box-shadow: 0 3px 10px #bbb;
+  --menu-box-shadow: 0 3px 10px #3e3e3e;
```

While this resolves the reported issue, the shadow is a lot darker than before.

I wonder whether the same effect on About Jenkins can be achieved by having a shadow color that's basically black + opacity. Something like this?

```suggestion
  --menu-box-shadow: 0 3px 10px rgba(0,0,0,0.3);
```

---

Need to add a space after "," or you can't run. I have made some change. please take a look.

---

> Need to add a space after "," or you can't run. I have made some change, please take a look.